--- gcompris-6.1/configure.in~	Tue Jul 27 20:33:46 2004
+++ gcompris-6.1/configure.in	Tue Jul 27 20:33:46 2004
@@ -114,7 +114,7 @@
 AM_ICONV
 
 dnl Add the languages which your application supports here.
-ALL_LINGUAS="am ar az ca cs de el en_CA en_GB es fi fr he hi hr hu it lt mk ml ms nl no pl pt pt_BR ro ru sk sl sr sr@Latn sv tr"
+ALL_LINGUAS="am ar az ca cs de el en_CA en_GB es fi fr he hi hr hu it lt mk ml ms nb nl pl pt pt_BR ro ru sk sl sr sr@Latn sv tr"
 
 dnl GCompris needs to know which locale are supported
 AC_SUBST(ALL_LINGUAS)