--- gcompris-6.3RC1/configure.in.orig	2004-09-24 09:01:40.440326200 +0200
+++ gcompris-6.3RC1/configure.in	2004-09-24 09:11:52.571268152 +0200
@@ -114,7 +114,7 @@
 AM_ICONV
 
 dnl Add the languages which your application supports here.
-ALL_LINGUAS="am ar az ca cs da de el en_CA en_GB es fi fr he hi hr hu it lt mk ml ms nl no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv tr wa"
+ALL_LINGUAS="am ar az ca cs da de el en_CA en_GB es fi fr he hi hr hu it lt mk ml ms nb nl pl pt pt_BR ro ru sk sl sq sr sr@Latn sv tr wa"
 
 dnl GCompris needs to know which locale are supported
 AC_SUBST(ALL_LINGUAS)
