diff -urN gcompris-6.1.orig/configure.in gcompris-6.1.new/configure.in
--- gcompris-6.1.orig/configure.in	2004-06-22 19:52:34.000000000 +0200
+++ gcompris-6.1.new/configure.in	2004-06-24 00:02:45.000000000 +0200
@@ -435,8 +435,8 @@
 dnl Autoconf output
 AC_OUTPUT([ gcompris.spec
 Makefile
-gcompris.desktop
-gcompris-edit.desktop
+gcompris.desktop.in
+gcompris-edit.desktop.in
 src/Makefile
 src/gcompris/Makefile
 src/gcompris/libgcompris-1.0.pc
diff -urN gcompris-6.1.orig/gcompris-edit.desktop.in gcompris-6.1.new/gcompris-edit.desktop.in
--- gcompris-6.1.orig/gcompris-edit.desktop.in	2003-10-07 13:14:36.000000000 +0200
+++ gcompris-6.1.new/gcompris-edit.desktop.in	2004-06-24 11:20:54.149725800 +0200
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=GCompris Editor
 _Comment=Editor for gcompris
-Exec=gcompris-edit
+Exec=gcompris_edit
 Icon=gcompris-edit.png
 Terminal=0
 Type=Application
